updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
 
 .Prerequisites
 
+* Have access to a conatiner registry in the disconnected environment to push and pull images. The container registry should be compatible with Docker registry API v2.
```

Comment:
nit: "conatiner" -> "container" and "disconnected" -> "restricted-network".

I dunno if "have `oc` installed" gets listed as a prerequisite or not.  Looks like we do list it explicitly in some places (e.g. [here][1]).

[1]: https://github.com/openshift/openshift-docs/blame/4abd206b42d38e5bd66423d780c4c900b395e8ff/modules/cli-configuring-completion.adoc#L13